scout/adapter/mongo/query.py

```diff
@@ -600,6 +600,22 @@ def secondary_query(self, query, mongo_query, secondary_filter=None):
                     }
                 )
 
+            if criterion == "swegen_freq":
+                swegen = query.get("swegen_freq")
+                if swegen == "-1":
```

Comment:
It is not clear from the filters with that -1 is a param that can be provided and what it means. Wouldn't be easier to add a checkbox that says "exclude variants on swegen"?

<img width="217" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/37616b73-4e3c-4e9d-9d6b-e14dfc23e25c">

Reply:
It's a Scout classic, been around since Robin & Måns, but perhaps it's not so well documented for the new users. 😬 Good call.

Reply:
Hm, actually it seems to only on gnomad, except for this one, and with the query itself since a long time already showing also non-existing values, this is no longer very useful. I think I'll remove both. Any second thoughts?

Reply:
No, go for it! 👍🏻